x/feemarket/keeper/integration_test.go

```diff
@@ -448,7 +446,7 @@ func (suite *IntegrationTestSuite) SetupTest(valMinGasPrice string, minGasPrice
 		Denom:  evmtypes.DefaultEVMDenom,
 		Amount: amount,
 	}}
-	privKey, address := suite.GenerateKey()
+	address := sdk.AccAddress(s.Address.Bytes())
```

Comment:
@yihuang it's not benchmark, ok to use const address?

Reply:
looks good